Week1/Solution 1/solution 4.py

```diff
@@ -0,0 +1,22 @@
+#4.Write a python script to enter any number, if it is integer number, then check its armstrong or not. Print appropriate message if it is not palindrom.

```

Comment:
Which is the better way to do a script using user define function or doing without using user define function?